fed/mosaic-output/src/main/resources/output_config.xsd

```diff
@@ -87,6 +88,13 @@
             </xs:documentation>
         </xs:annotation>
     </xs:element>
+    <xs:element name="decimalSeperator" type="xs:string">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                The seperator for floating-point numbers.
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
```

Comment:
Would be good to add the new config option to existing `output_config.xml` files, e.g. Barnim scenario.